source/help/getting-started/managing-members.rst

```diff
@@ -1,77 +1,97 @@
 
-#### Direct Invites  
+Direct Invites
+~~~~~~~~~~~~~~
 
 Direct invites are invitation emails sent from your teams server directly to the invited members email address. A link within the invitation email directs the users to an account creation page.
 
-From the three dot **Main Menu** in the Mattermost header, select **Invite New Member**. Enter a new members email address along with their first and last name. Click **Add another** to send more direct invites from the same dialog.
+From the three dot **Main Menu** in the Mattermost header, select **Invite New Member**. Enter a new member's email address along with their first and last name. Click **Add another** to send more direct invites from the same dialog.
 
-#### Team Invite Link  
+Team Invite Link
+~~~~~~~~~~~~~~~~
 
-The team invite link is a unique URL that directs users to an account creation page for your team. Your team invite link can be used by any number of users and does not change unless it is re-generated by a Team or System Admin via **Team Settings** > **General** > **Invite Code**. For example, the team invite link can be included in a company wide email to invite all employees to join a Mattermost team.
+The team invite link is a unique URL that directs users to an account creation page for a team. A team invite link can be used by any number of users and does not change unless it is re-generated by a Team Admin or System Admin via **Team Settings > General > Invite Code**. For example, the team invite link can be included in a company wide email to invite all employees to join a Mattermost team.
 
 From the three dot **Main Menu** in the Mattermost header, select **Get Team Invite Link**. Copy the link and share it with users you'd like to invite to the team.
 
-## User Roles  
+User Roles
+----------
 
-There are four types of user roles with different permission levels in Mattermost: System Admins, Team Admins, Members and Inactive accounts. To view a list of users on the team and what their roles are, Team Administrators can go to the three dot **Main Menu** and select **Manage Members**.  
+There are four types of user roles with different permission levels in Mattermost: System Admins, Team Admins, Members and Inactive accounts. To view a list of users on the team and what their roles are, Team Administrators can go to the three dot **Main Menu** and select **Manage Members**.
 
-#### Member 
+Member
+~~~~~~
 
 This is the default role given to users when they join a team. Members have basic permissions on the Mattermost team.
 
-#### Team Admin 
+Channel Admin
+~~~~~~~~~~~~~
+
+The person who creates a channel is assigned the Channel Admin role for that channel. People with the Channel Admin role have the following privileges:
+
+- Ability to assign the Channel Admin role to other members of the channel.
+- Ability to remove the Channel Admin role from other holders of the Channel Admin role.
+- Ability to remove members from the channel.
```

Comment:
We should pronbably also mention:

"Depending on your system configuration, Channel Admins can be granted special permissions by the System Admin to rename and delete channels."